developerGuide.t2t

```diff
@@ -374,26 +369,76 @@ When NVDA receives input, it looks for a matching gesture binding in a particula
 Once a gesture binding is found, the script is executed and no further bindings are used, nore is that particular gesture passed on automatically to the Operating System.
 
 The order for gesture binding lookup is:
+- The user specific gesture map
+- The locale specific gesture map
+- The braille display driver specific gesture map
 - Loaded Global Plugins
 - App Module of the active application
 - Tree Interceptor of the NVDA Object with focus if any; e.g. a virtualBuffer
 - NVDA Object with focus
 - Global Commands (built in commands like quitting NVDA, object navigation commands, etc.)
 -
 
-You should specify a description of the script in the function's docstring which describes the command for users.
+You should also specify a description for the script which describes the command for users.

```

Comment:
I'm worried about the ordering here. We tell the reader all the things they must specify, but the info on how to specify them is in a later subsection. I think this needs to be reordered a bit so that the text describing what properties should specify (and what they mean) is interwoven with the info about declaring a script with the decorator.